writing-visualising/dashboards.qmd

```diff
@@ -52,7 +52,7 @@ The following sections cover the considerations in more detail, though when deve
 
 ---
 
-All teams developing dashboards to accompany Official Statistics should contact the [Statistics Development Team](mailto:statistics.development@education.gov.uk) for advice on the level of approval required as there may be times when HoP approval is needed in addition to team / unit leader approval. Approvals should be sent to the Statistics Development Team when asking for a new dashboard to be hosted on shinyapps.io.
+All teams developing dashboards to accompany Official Statistics should contact the [explore education statistics platforms team](mailto:exp@education.gov.uk) for advice on the level of approval required as there may be times when HoP approval is needed in addition to team / unit leader approval. Approvals should be sent to the Statistics Development Team when asking for a new dashboard to be hosted on shinyapps.io.
```

Comment:
this email address looks like a bit is missing?

Reply:
Amazing, no idea how I did that! Fixed in next commit